src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -631,7 +631,7 @@ public static synchronized String[] getAvailableIDs() {
      * time zone.
      *
      * <ul>
-     * <li>Use the {@code user.timezone} property value as the default
+     * <li>Use the {@systemProperty user.timezone} property value as the default
```

Comment:
I'd only modify this location and keep others intact. Otherwise, the search would show all locations which may not best serve the user's needs.

Reply:
Good point, I reverted the rest, so that only this first instance is tagged as _@systemProperty_

Reply:
Like Naoto notes, my understanding of it is that the `@systemProperty` should appear only once at the place where the semantics of that system property is being defined. This mail, which was sent when this tag was introduced, has more details https://mail.openjdk.org/pipermail/core-libs-dev/2018-November/056653.html.

I think using it here is fine, but you may want to check how it "reads" on the generated javadoc index page for system properties.

Reply:
> but you may want to check how it "reads" on the generated javadoc index page for system properties.

For clarity, I meant the page where you will see all the system properties - there's a "Index" on top of the javadoc API page https://docs.oracle.com/en/java/javase/19/docs/api/index.html which leads to a page which has a link to the "System Properties" page, which should lead to https://docs.oracle.com/en/java/javase/19/docs/api/system-properties.html

Reply:
Hi Jai,

I read the article and have a better understanding of when _@systemProperty_ should be used and the supporting features.

I wasn't aware such a [page](https://docs.oracle.com/en/java/javase/19/docs/api/system-properties.html) existed, I checked that _user.timezone_ was displayed and linked to _java.util.TimeZone.getDefault()_ properly.

Thanks for the help!